src/modules/logcollector/src/journald_reader.cpp

```diff
@@ -0,0 +1,77 @@
+#include "journald_reader.hpp"
+#include <logger.hpp>
+#include <sstream>
+
+using namespace logcollector;
```

Comment:
```suggestion
namespace logcollector
{
...
}
```
Why don't you do this?